source/constructs/lib/common-resources/custom-resources/custom-resource-construct.ts

```diff
@@ -84,6 +84,7 @@ export class CustomResourcesConstruct extends Construct {
                 "s3:GetObject",
                 "s3:PutObject",
                 "s3:ListBucket",
+                "s3:PutBucketOwnershipControls",
```

Comment:
why is this needed by the lambda, is that lambda creating the logging bucket?

Reply:
yes, that was the cause of the access denied error